code/modules/mining/abandoned_crates.dm

```diff
@@ -12,7 +12,7 @@
 	var/qdel_on_open = FALSE
 	tamperproof = 90
 
-/obj/structure/closet/crate/secure/loot/Initialize()
+/obj/structure/closet/crate/secure/loot/open(mob/living/user)
 	. = ..()
 	var/list/digits = list("1", "2", "3", "4", "5", "6", "7", "8", "9", "0")
```

Comment:
this code needs to be Initialize or the create will not be openable

Reply:
the code that should be in open starts with
`var/loot = rand(1,100) //100 different crates with varying chances of spawning`